.mailmap

```diff
@@ -1007,6 +1007,7 @@ Mihir Wadwekar <m.mihirw@gmail.com>
 Mike Boyle <boyle@astro.cornell.edu>
 Mikel Rouco <mikel.mrm@gmail.com>
 Mikhail Remnev <maremnev@gmail.com> Mikhail Remnev <141655736+maremnev@users.noreply.github.com>
+Mikoláš Janota <mikolas.janota@gmail.com>
```

Comment:
Change this to:
```
Mikoláš Janota <mikolas.janota@gmail.com> mikolas <mikolas@gmail.com>
```
The first name and email will go in the AUTHORS file. The second name and email is what is recorded in your commit:
https://github.com/sympy/sympy/commit/9b26f4ebe7f8c855557f4ee92bbf872480353811.patch

Reply:
> Change this to:
> 
> ```
> Mikoláš Janota <mikolas.janota@gmail.com> mikolas <mikolas@gmail.com>
> ```
> 
> The first name and email will go in the AUTHORS file. The second name and email is what is recorded in your commit: https://github.com/sympy/sympy/commit/9b26f4ebe7f8c855557f4ee92bbf872480353811.patch

yeah, the mikolas@gmail.com somehow got in and it's wrong, I don't know how to change it in the commit

Reply:
Looks like you've fixed it now.